src/stactools/landsat/utils.py

```diff
@@ -98,3 +98,34 @@ def handle_antimeridian(item: Item, antimeridian_strategy: Strategy) -> None:
         item.geometry = Polygon(merged_coords)
 
     antimeridian.fix_item(item, antimeridian_strategy)
+
+
+def round_coordinates(item: Item, precision: int = 6) -> None:
```

Comment:
```suggestion
def round_coordinates(item: Item, precision: int = 6) -> Item:
```

It's undocumented, but I've started preferring a "fluent" interface where we both modify the item in place and return the modified item. This allows the function to be used in assignment-style setups (`item = round_coordinates(create_item("file.tif"))`) as well as mutation setups (`round-coordinates(item)`).

As a general note, this function does seem useful enough to push into **stactools** at a later date.

Reply:
Yeah, we've discussed this concept. I did not include it in order to be consistent with the `handle_antimeridian` function. But I'm fine with changing it. I updated both `round_coordinates` and `handle_antimeridian` to be fluent, for consistency.